docs/how_to/integrations/vector_stores.md

```diff
@@ -206,6 +206,7 @@ response = index.query("What did the author do growing up?")
 
 **Milvus Index Construction/Querying**
```

Comment:
@filip-halt would you like to clarify here too?

Reply:
Let me do that. Wasnt sure how much I was allowed to add in that section in terms of descriptions and extra info.